README.md

```diff
@@ -1,6 +1,6 @@
 # Automata
 
-*Copyright 2016-2022 Caleb Evans*  
+*Copyright 2016-2022 Caleb Evans*
```

Comment:
@eliotwrobson Just a note for the future: your editor seems to keep removing the two trailing spaces that is valid in Markdown to insert a line break. Make sure you have the editorconfig extension installed in your editor.